adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/api/rest/v1/AdapterEndpoint.java

```diff
@@ -16,32 +16,58 @@
 @RequestMapping("/")
 public class AdapterEndpoint {
 
-  private final Adapter adapter;
+    private final Adapter adapter;
```

Comment:
Why did you change the indentation to 4 spaces? According to the `.editorconfig` file in the root of the repository, all files should be intended with two spaces.

Reply:
I blame IntelliJ!
Thanks for pointing it out, I will fix that